xapi-rrdd-plugin.opam

```diff
@@ -0,0 +1,9 @@
+opam-version: "1.2"
+maintainer: "john.else@citrix.com"
```

Comment:
shall we change the name to someone real here?

Reply:
That's a real name, although he left citrix :-)
I am updating it with `xen-api@lists.xen.org` as we did for other opam files.